app/javascript/components/projects/ProjectView.jsx

```diff
@@ -54,9 +56,14 @@ class ProjectView extends React.Component {
     } = this.props;
 
     var org_img = <span></span>;
-    if (this.props.org_image_url) {
+    //if (this.props.org_image_url)
+    if (0) {
```

Comment:
I think we should remove this entire if statement for now and change it later once we have the project image in the model.